cli/dts/lib.deno.shared_globals.d.ts

```diff
@@ -1589,6 +1631,11 @@ interface ProgressEvent<T extends EventTarget = EventTarget> extends Event {
   readonly total: number;
 }
 
+declare var ProgressEvent: {
+  prototype: ProgressEvent;
+  new(type:string, eventInitDict?: ProgressEventInit): ProgressEvent;
```

Comment:
@PaulThompson tests are failing because `ProgresEventInit` interface is not defined